README.md

```diff
@@ -77,36 +86,103 @@ public class StreamsBootstrapApplication extends KafkaStreamsApplication {
 }
 ```
 
-The following configuration options are available (can be passed per arguments or environment variables):
+The following configuration options are available:
+
+- `--brokers`: List of Kafka brokers (comma-separated) (**required**)
+
+- `--schema-registry-url`: The URL of the schema registry (**required**)
+
+- `--input-topics`: List of input topics (comma-separated)
+
+- `--input-pattern`: Pattern of input topics
+
+- `--output-topic`: The output topic
+
+- `--error-topic`: A topic to write errors to
+
+- `--streams-config`: Kafka Streams configuration (`<String=String>[,<String=String>...]`)
+
+- `--extra-input-topics`: Additional named input topics if you need to specify multiple topics with different message
+  types (`<String=String>[,<String=String>...]`)
+
+- `--extra-input-patterns`: Additional named input patterns if you need to specify multiple topics with different
+  message types (`<String=String>[,<String=String>...]`)
+
+- `--extra-output-topics`: Additional named output topics if you need to specify multiple topics with different message
+  types (`String=String>[,<String=String>...]`)
+
+- `--clean-up`: Whether the state of the Kafka Streams app, i.e., offsets and state stores and auto-created topics,
```

Comment:
```suggestion
- `--clean-up`: Whether the state of the Kafka Streams app, i.e., offsets and state stores and internal topics,
```

Reply:
I wanted to highlight that it only affects auto-created topics. A topic that the user creates but is only used internally could also fall under your description

Reply:
Ah, good point 👍